activestorage/lib/active_storage/attached.rb

```diff
@@ -28,8 +28,6 @@ def create_blob_from(attachable)
           ActiveStorage::Blob.create_after_upload!(attachable)
         when String
           ActiveStorage::Blob.find_signed(attachable)
-        else
-          nil
```

Comment:
The presence of this `else` was previously discussed in this comment:
https://github.com/rails/rails/commit/b7cc003aa0aada594cb18ab80c13c13c75bcd389#r27841179